src/pages/AgentNew.tsx

```diff
@@ -106,16 +106,8 @@ export default function AgentNew() {
               <Form.Control
                 type="text"
                 placeholder={t('agent.form.firstName')}
```

Comment:
Placeholder can be removed (it isn't used for floating label inputs).

The label on line 103 should be 
```javascript
  label={`${t('agent.form.firstName')} ${t('ui.optional')}`}
```

Reply:
Also removing placeholder from last name in accordance with this.